config/initializers/shoryuken.rb

```diff
@@ -11,4 +11,5 @@
 
 Shoryuken.configure_server do |_config|
   Shoryuken::Logging.logger = Rails.logger
+  Shoryuken::Logging.logger.level = :info
```

Comment:
What does Shoryuken do?

Reply:
[Shoryuken](https://github.com/ruby-shoryuken/shoryuken) is the gem that hooks our workers up to the SQS queues.  So any messages on those queues get picked up and processed.